csharp/msbuild/zeroc.ice.net.nuspec

```diff
@@ -13,5 +13,6 @@
         <description>Ice C#/.NET SDK. Ice is a comprehensive RPC framework that helps you network your software with minimal effort.</description>
```

Comment:
We should update this description. 

The [Ice framework](https://github.com/zeroc-ice/ice) provides everything you need to build networked applications, including RPC, pub/sub, server deployment, and more.

Reply:
Fixed as suggested